backend/src/LibExecution/ProgramTypes.fs

```diff
@@ -588,7 +586,7 @@ module UserFunction =
     { tlid : tlid
       name : FnName.UserProgram
       typeParams : List<string>
-      parameters : List<Parameter>
+      parameters : NEList<Parameter>
```

Comment:
If there's no parameters, than presumably this will be a constant? Though what if there's typeParams and no params? What would we do with that?